app/src/main/java/com/yml/womensafety/navigationdrawer/youtube/SelfDefenseVideoFragment.kt

```diff
@@ -18,21 +20,17 @@ class SelfDefenseVideoFragment : Fragment(R.layout.fragment_self_defense_video)
     override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
         super.onViewCreated(view, savedInstanceState)
 
-        youTubeViewModel = ViewModelProvider(this).get(YouTubeViewModel::class.java)
+        youTubeViewModel = ViewModelProvider(this).get(YouTubeViewModel()::class.java)
         youTubeViewModel.initializeRepository()
+        videoRecyclerView = view.findViewById(R.id.recyclerViewVideos)
```

Comment:
Some places I see synthetic import and some places findViewById , Why this difference ?